lib/audio/songs.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:audioplayers/audioplayers.dart';
+
+class Songs {
+  static AssetSource mainMenuAmbient = AssetSource('audio/main_menu_theme.mp3');
+  static AssetSource clickerTheme = AssetSource('audio/clicker_theme.mp3');
+  static AssetSource catcherTheme = AssetSource('audio/catcher_theme.mp3');
+  static AssetSource finderTheme = AssetSource('audio/finder_theme.mp3');
+}
```

Comment:
Flutter_gen supports mp3, so just update assets there is no need for this class. 
For reference: [link](https://github.com/VeryGoodOpenSource/very_good_flame_game/blob/main/brick/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D/lib/gen/assets.gen.dart)

Reply:
Done